lib/account/device_linking/authorize_device_via_email_pin.dart

```diff
@@ -74,6 +74,8 @@ class AuthorizeDeviceViaEmailPin extends StatelessWidget {
     try {
       context.loaderOverlay.show();
       await sessionModel.validateDeviceRecoveryCode(code);
+    //once code is validated, approve the device
+      await sessionModel.approveDevice(code);
```

Comment:
@atavism I think we need to approve the device once the code is validated correct?